virt/backup_restore/virt-disaster-recovery.adoc

```diff
@@ -0,0 +1,14 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-disaster-recovery"]
+= Disaster recovery
```

Comment:
Should this match the "About disaster recovery" title used in left-nav, or vice versa?